### PR TITLE
update readme with note on when new fragments are allowed

### PR DESCRIPTION
**Pull request by katiegoines**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ When linking to other pages within the docs.amplify.aws site, do not use relativ
 
 To incorporate new platform-specific content within a page, please use [Inline Filters](https://github.com/aws-amplify/docs/blob/main/Readme.md#inline-filters).
 
+Please note: we do allow fragments in some cases where content is intended to be reused and cannot be  via InlineFilters
```
Comment (timngyn) [inline Readme.md:94]: Removing extra space:

```suggestion
Please note: we do allow fragments in some cases where content is intended to be reused and cannot be via InlineFilters
```

**Review by hbuchel — APPROVED** [review]

**Review by esalacupnamde — APPROVED** [review]
Great job on this 🏅

**Review by chrisbonifacio — APPROVED** [review]

**Comment by esalacupnamde** [comment]
@litwicki AWSOMEEEE!

**Comment by esalacupnamde** [comment]
May I should Pursue COmmunications instead of Computer science deggreeee @litwicki , Any thoughts?

**Pull request merged**